db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `state_machine_name` VARCHAR(255) NOT NULL,
```

Comment:
I really didn't understand.
If we are linking two tables, why not use the PK of one as FK in the other?